app/main.py

```diff
@@ -1,3 +1,24 @@
+import math
+
+
 class OnlineCourse:
-    # write your code here
-    pass
+    def __init__(self,
+                 name: str,
+                 description: str,
+                 weeks: int):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days: int):
+        return math.ceil(days / 7)
+
+    @classmethod
+    def from_dict(cls, course_dict):
+        weeks = cls.days_to_weeks(course_dict['days'])
```

Comment:
It would be better to use `OnlineCourse.days_to_weeks` here. You will discuss this case in the future lectures on OOP

Reply:
ok